app/AppGroup.m

```diff
@@ -101,11 +101,11 @@
                                                                       error:nil];
 }
 
-NSArray<NSString *> *CurrentAppGroups() {
+NSArray<NSString *> *CurrentAppGroups(void) {
     return AppEntitlements()[@"com.apple.security.application-groups"];
 }
 
-NSURL *ContainerURL() {
+NSURL *ContainerURL(void) {
```

Comment:
Pretty sure (void) isn't needed for function definitions, only prototypes

Reply:
It is not, although trying to call it incorrectly invokes undefined behavior

Reply:
The idea is you need it on the prototype so you get an error when you call it wrong, but you don't need it on the definition

Reply:
Apparently defining it as such creates a 0-argument prototype as well, and then trying to call a function that has a prototype defined with the wrong number of arguments is undefined.

Reply:
Weird but ok